Index: xen-orchestra/packages/xo-web/src/xo-app/index.js
===================================================================
--- xen-orchestra.orig/packages/xo-web/src/xo-app/index.js
+++ xen-orchestra/packages/xo-web/src/xo-app/index.js
@@ -132,14 +132,9 @@ export default class XoApp extends Component {
     }
   }
 
-  dismissSourceBanner = () => this.setState({ dismissedSourceBanner: true })
-
   componentDidMount() {
     this.refs.bodyWrapper.style.minHeight =
       this.refs.menu.getWrappedInstance().height + 'px'
-    if (+process.env.XOA_PLAN === 5) {
-      this.displayOpenSourceDisclaimer()
-    }
   }
 
   _shortcutsHandler = (command, event) => {
@@ -225,20 +220,6 @@ export default class XoApp extends Component {
                   </Link>
                 </div>
               )}
-              {plan === 'Community' && !this.state.dismissedSourceBanner && (
-                <div className='alert alert-danger mb-0'>
-                  <a
-                    href='https://xen-orchestra.com/#!/xoa?pk_campaign=xo_source_banner'
-                    rel='noopener noreferrer'
-                    target='_blank'
-                  >
-                    {_('disclaimerText3')}
-                  </a>
-                  <button className='close' onClick={this.dismissSourceBanner}>
-                    &times;
-                  </button>
-                </div>
-              )}
               <div style={CONTAINER_STYLE}>
                 <Shortcuts
                   name='XoApp'

