Index: xen-orchestra/packages/xo-web/src/xo-app/index.js
===================================================================
--- xen-orchestra.orig/packages/xo-web/src/xo-app/index.js
+++ xen-orchestra/packages/xo-web/src/xo-app/index.js
@@ -222,17 +222,6 @@ export default class XoApp extends Compo
                   </Link>
                 </div>
               )}
-              {+process.env.XOA_PLAN === 5 && (
-                <div className='alert alert-danger mb-0'>
-                  <a
-                    href='https://xen-orchestra.com/#!/xoa?pk_campaign=xo_source_banner'
-                    rel='noopener noreferrer'
-                    target='_blank'
-                  >
-                    {_('disclaimerText3')}
-                  </a>
-                </div>
-              )}
               <div style={CONTAINER_STYLE}>
                 <Shortcuts
                   name='XoApp'
